crates/openssl-src-111/src/deterministic_rand.c

```diff
@@ -1,25 +1,37 @@
 // based on https://stackoverflow.com/a/7510354
 #include <openssl/rand.h>
+#include <stdint.h>
 #include <stdlib.h>
 
-static uint64_t seed = 42;
+#ifndef thread_local
+// since C11 the standard include _Thread_local
+#if __STDC_VERSION__ >= 201112 && !defined __STDC_NO_THREADS__
+#define thread_local _Thread_local
+
+// note that __GNUC__ covers clang and ICC
+#elif defined __GNUC__ || defined __SUNPRO_C || defined __xlC__
+#define thread_local __thread
```

Comment:
Could we use that to solve the second item of #307?! @aeyno

Reply:
Done, see #310.